### PR TITLE
feat: reject street name

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/StreetNameRegistry.Producer/ProducerProjections.cs [hunk 1]
```diff
@@ -134,7 +134,7 @@ public ProducerProjections(IConfiguration configuration)
             {
                 await Produce(message.Message.StreetNameId, message.Message.ToContract(), ct);
             });
-            
+
```
Comment (lvermeulen) [inline src/StreetNameRegistry.Producer/ProducerProjections.cs:137]: remove the extra blank line

File: src/StreetNameRegistry.Producer/ProducerProjections.cs [hunk 2]
```diff
@@ -164,7 +164,7 @@ public ProducerProjections(IConfiguration configuration)
...
```
Comment (lvermeulen) [inline src/StreetNameRegistry.Producer/ProducerProjections.cs:167]: remove the extra blank line

File: src/StreetNameRegistry.Producer/ProducerProjections.cs [hunk 3]
```diff
@@ -199,6 +199,11 @@ public ProducerProjections(IConfiguration configuration)
             {
                 await Produce(message.Message.PersistentLocalId, message.Message.ToContract(), ct);
             });
+
```
Comment (lvermeulen) [inline src/StreetNameRegistry.Producer/ProducerProjections.cs:202]: remove the extra blank line

File: src/StreetNameRegistry/Municipality/Exceptions/StreetNameStatusPreventsRejectionException.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+namespace StreetNameRegistry.Municipality.Exceptions
+{
+    public class StreetNameStatusPreventsRejectionException : StreetNameRegistryException
```
Comment (lvermeulen) [inline src/StreetNameRegistry/Municipality/Exceptions/StreetNameStatusPreventsRejectionException.cs:3]: please conform to Serializable specification

Comment (jvandaal) [reply src/StreetNameRegistry/Municipality/Exceptions/StreetNameStatusPreventsRejectionException.cs:3]: Are you referring to the `[Serializable]` attribute? None of the exceptions in the project are (directly) decorated by the attribute. They do inherit from abstract base class `DomainException` which has the attribute. Do you want us to start decorating the exception(s) with the attribute?

File: src/StreetNameRegistry.Producer/ProducerProjections.cs (hunk 3 above)
Comment (jvandaal) [reply src/StreetNameRegistry.Producer/ProducerProjections.cs:202]: There's only a single blank line between the two statements.

File: src/StreetNameRegistry.Producer/ProducerProjections.cs (hunk 2 above)
Comment (jvandaal) [reply src/StreetNameRegistry.Producer/ProducerProjections.cs:167]: There's only a single blank line between the two statements.

File: src/StreetNameRegistry.Producer/ProducerProjections.cs (hunk 1 above)
Comment (jvandaal) [reply src/StreetNameRegistry.Producer/ProducerProjections.cs:137]: There's only a single blank line between the two statements.

File: src/StreetNameRegistry/Municipality/Exceptions/StreetNameStatusPreventsRejectionException.cs (hunk 4 above)
Comment (lvermeulen) [reply src/StreetNameRegistry/Municipality/Exceptions/StreetNameStatusPreventsRejectionException.cs:3]: I will do it

**Pull request merged**